.vscode/extensions.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
We can't remove this

Reply:
My bad, I confused them with some files that change editor colors.